x-pack/plugins/serverless_search/public/application/components/pipeline_panel.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiThemeProvider,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiSpacer,
+  EuiText,
+  EuiImage,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { useAssetBasePath } from '../hooks/use_asset_base_path';
+
+export const PipelinePanel: React.FC = () => {
+  const assetBasePath = useAssetBasePath();
+
+  return (
+    <EuiThemeProvider colorMode="dark">
+      <EuiPanel paddingSize="xl">
+        <EuiFlexGroup alignItems="flexStart" justifyContent="flexStart">
+          <EuiFlexItem grow={false}>
+            <EuiImage alt="cluster" src={`${assetBasePath}/cluster.svg`} />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiTitle size="s">
+              <h3>
+                {i18n.translate('xpack.serverlessSearch.pipeline.overview.dataEnrichment.title', {
+                  defaultMessage: 'Enrich Data',
+                })}
+              </h3>
+            </EuiTitle>
+            <EuiSpacer size="s" />
```

Comment:
I think we may want to wrap these in a `FlexGroup` so we can do a gutter size between each of these rows to match the spacing in the design a bit closer.